### ISSUE TITLE
A11Y. "Configuration” and “Overview” tabs not focusable

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/spartacus#11240]
**Setup**
3.1.0.RC2 Server Links
 CCV2 (CX 2011.1):
-- b2c - https://spartacusstore.cg79x9wuu9-eccommerc1-p5-public.model-t.myhybris.cloud/electronics-spa/en/USD/
-- b2b - https://b2bspastore.cg79x9wuu9-eccommerc1-p5-public.model-t.myhybris.cloud/powertools-spa/en/USD/

 Linux (CX 2005.6):
-- b2c - http://spartacus-dev2.eastus.cloudapp.azure.com:4200/electronics-spa/en/USD/
-- b2b - http://spartacus-dev2.eastus.cloudapp.azure.com:5200/powertools-spa/en/USD/

**Steps to Reproduce (Using Keyboard Only):**

1. Navigate to B2C Storefront
2. Search for HOME
3. From product listing page, select Home Theatre product.
4. Add to cart (without making any configuration changes to the product)
5. Attempt to navigate to Configuration tab using keyboard (tab key mostly)

**Expected Results:** 
5. Able to switch tabs between 'Configuration' and 'Overview'

**Actual Results:** 
5. Switching tabs not possible.

**Comment by giancorderoortiz** [comment]
![Screen Shot 2021-02-24 at 12 24 47 PM](https://user-images.githubusercontent.com/46171897/109043281-2b1b0200-769f-11eb-878a-a83729ad0432.png)

**Comment by giancorderoortiz** [comment]
@developpeurweb please confirm steps to reproduce.

**Comment by developpeurweb** [comment]
> @developpeurweb please confirm steps to reproduce.

Confirming steps as described.
Currently not focusable.
While addressing this issue pls note these should be action-links, meaning **buttons** styled as links since the action happens on the same page. See similar solution used in Order History.

**Comment by steinsebastian** [comment]
Will be handled with v1.

**Comment by giancorderoortiz** [comment]
@Platonn @Larisa-Staroverova to be fixed in 3.3 ?

**Comment by Larisa-Staroverova** [comment]
We will take care of it
